README.md

```diff
@@ -12,11 +12,15 @@ This DSL relies on the Ruby [pact-mock_service][pact-mock-service] gem to provid
 * Google users group: https://groups.google.com/forum/#!forum/pact-support
 * Twitter: [@pact_up](https://twitter.com/pact_up)
 
-### Getting Started (with Karma, Jasmine and the pact-mock_service gem)
+### Installing pact-mock-service
 
-1. Install the [pact-mock_service](https://github.com/bethesque/pact-mock_service) ruby gem
+* If on Windows, please refer to the [Installing pact-mock-service on Windows](https://github.com/bethesque/pact-mock_service/wiki/Installing-the-pact-mock_service-gem-on-Windows) *
 
-   The easiest way is to add `gem 'pact-mock_service', '~> 0.2.3.pre.rc1'` to your `Gemfile` and run `bundle install`
+1. You must install [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) first.
```

Comment:
That's like me claiming that I don't need to follow good node practice because I'm not a node developer :P
The problems caused by not using Gemfiles and bundle exec happen to people who write Ruby, and people who don't write Ruby. When they happen to people who don't write Ruby, then it's even harder for them to diagnose the cause. If there is a backwards incompatible change to the mock server, and you upgrade one project but not another, then one of your projects is going to break.